app/services/doc_auth/socure/requests/docv_result_request.rb

```diff
@@ -6,7 +6,8 @@ module Requests
       class DocvResultRequest < DocAuth::Socure::Request
         attr_reader :document_capture_session_uuid, :biometric_comparison_required
 
-        def initialize(document_capture_session_uuid:, biometric_comparison_required: false)
+        def initialize(document_capture_session_uuid:,
+                       biometric_comparison_required: false)
```

Comment:
```suggestion
        def initialize(document_capture_session_uuid:, biometric_comparison_required: false)
```
undo for readability